src/go/cmd/pkg.go

```diff
@@ -27,6 +29,7 @@ var ErrNoPackages = errors.New("no packages found")
 
 // Pkg represents a Go package.
 type Pkg struct {
+	ModulePath  string
```

Comment:
Nit: Is there a reason why this is exported compared to the other fields?  If not, can you please unexport it for consistency.